app/models/jobsModel.js

```diff
@@ -0,0 +1,22 @@
+const db = require('../database/dbConfig.js')
+
+/**
+ * Inserts a job into the database.
+ * scheduled_time is an optional field.
+ * 
+ * @param {"class_id" : integer, 
+ *         "scheduled_time" : DateTime, 
```

Comment:
DateTime is actually not a type in js, it is more clear to just change this to Date, a Date object in js contains date and time